web/test/integration/project/dashboard.feature

```diff
@@ -5,17 +5,15 @@ Feature:  Project dashboard page
     When the "/projects/test-public-project-minimum" page is open
     Then the text "You do not have any monitors. Please create monitor" is not in the element ".helper"
     And the text "You do not have any repositories. Please connect repository" is not in the element ".helper"
-    And the text "No monitors added to this project yet" is not in the element ".helper"
+    And the text "No monitors added to this project yet" is in the element ".helper"
```

Comment:
This should be `not` for `guest`, this is an project owner message

Reply:
Is it ? Should it not visible to guest ?

Reply:
Yes it should not be visible to anyone apart from the project owner

Reply:
Then its correct message for guest.

Reply:
No it is not correct, because only the owner should see this message. We need to hide the message for everyone else (guest or another member)

Reply:
I just checked the code and you are right, it is right as there is another one for the owner. I dont like it, but it is something we can change in the future